apps/src/gamelab/GameLab.js

```diff
@@ -411,9 +360,14 @@ GameLab.prototype.execute = function() {
     if (!this.JSInterpreter.initialized()) {
       return;
     }
-    this.drawFunc = this.JSInterpreter.findGlobalFunction('draw');
-    this.setupFunc = this.JSInterpreter.findGlobalFunction('setup');
-    this.mousePressedFunc = this.JSInterpreter.findGlobalFunction('mousePressed');
+
+    ['draw', 'setup'].concat(this.p5eventNames).forEach(function (eventName) {
```

Comment:
What am I saying?  Replace `['draw', 'setup'].concat(this.p5eventNames)` with `this.p5specialFunctions`.